src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -250,9 +250,9 @@ private void testAddAction() throws Exception {
         feedbackPage.clickNeverVisibleTimeButton();
 
         //verify that timeFrameTable, instructions and ResponseVisTable are all hidden
-        feedbackPage.verifyHidden(By.id("timeFramePanel"));
-        feedbackPage.verifyHidden(By.id("responsesVisibleFromColumn"));
-        feedbackPage.verifyHidden(By.id("instructionsRow"));
+        assertTrue(feedbackPage.verifyHidden(By.id("timeFramePanel")));
```

Comment:
Yes, they should be renamed too. `verify` or `assert` prefix should be used only if the method contains asserts.